clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -1150,8 +1150,8 @@ public void testRefreshOffsetLoadInProgress() {
 
         subscriptions.assignFromUser(singleton(tp));
         subscriptions.needRefreshCommits();
-        client.prepareResponse(offsetFetchResponse(tp, Errors.GROUP_LOAD_IN_PROGRESS.code(), "", 100L));
-        client.prepareResponse(offsetFetchResponse(tp, Errors.NONE.code(), "", 100L));
+        client.prepareResponse(offsetFetchResponse(tp, Errors.NONE.code(), "", 100L, Errors.GROUP_LOAD_IN_PROGRESS.code()));
```

Comment:
Sounds good to me. I'll work on the rest of items you found in the meantime that PR gets merged. Then we can revisit this piece.